### PR TITLE
Updated README.md in CDC_BACKEND directory

### LINKED ISSUE #110: Api Documentation - AdminViews
Go through the different Admin Views and update the [README](https://github.com/CDC-IITDH/cdc-placement-website-backend/tree/main/CDC_Backend#readme) at ./CDC_Backend/README.md
 
- [x] markStatus/
- [x] getDashboard/
- [x] updateDeadline/
- [x] updateOfferAccepted/
- [ ] updateEmailVerified
- [ ] updateAdditionalInfo/
- [x] getApplications/
- [x] submitApplication/
- [x] generateCSV/
- [x] addPPO/
- [x] getStudentApplication/

### PR DESCRIPTION
**Pull request by jp-karthik**
closes #110

### CODE REVIEW

File: CDC_Backend/README.md [hunk 1]
```diff
@@ -617,106 +602,97 @@ Response is a Json with these fields
...
+      "tentative_no_of_offers": 5,
+      "other_requirements": "above 8 cpi",
+      "additional_info": ["School", "Home Town"],
+      "email_verified": false,
+      "offer_accepted": null,
+      "deadline_date": "2021-12-04",
+      "created_at": "2021-12-02T20:12:21+05:30"
+    }
+  ],
+  "previous": []
```
Comment (gowtham3105) [inline CDC_Backend/README.md:695]: ` New` field is missing

**Review by gowtham3105 — DISMISSED** [review]

File: CDC_Backend/README.md [hunk 2]
```diff
@@ -917,10 +891,236 @@ Response is a Json with these fields
...
+> Authorization: "Bearer {tokenID}"
+
+### Response
+
+Response is a Json with these fields
+
+```json
+{
+  "action": "Get Student Application",
+  "application_found": "true",
```
Comment (karthikmurakonda) [inline CDC_Backend/README.md:1101]: add another response example where `"application_found":"false"`

File: CDC_Backend/README.md [hunk 3]
```diff
@@ -917,10 +891,236 @@ Response is a Json with these fields
...
+  "application_info": {
+    "id": "F28IRGGMC3RP8Y8",
+    "additional_info": ,
```
Comment (karthikmurakonda) [inline CDC_Backend/README.md:1104]: additional info is a json objects with keys as entered by company for example company may 12th marks i.e `"additional_info":{"12marks":98}`.

File: CDC_Backend/README.md [hunk 4]
```diff
@@ -917,10 +891,236 @@ Response is a Json with these fields
...
+    "resume": "resume_link"
+  },
+  "student_details": {
+    "name": "John",
+    "batch": "2020",
+    "branch": "MMAE",
+    "resume_list":
```
Comment (karthikmurakonda) [inline CDC_Backend/README.md:1111]: its an array of json with keys name and link

File: CDC_Backend/README.md [hunk 5]
```diff
@@ -917,10 +891,236 @@ Response is a Json with these fields
...
+```
+
+### Status Codes
+
+The possible responses for this api request are as follows
+
+| Status Codes    | Possible Messages      |
+| --------------- | ---------------------- |
+| 200 OK          | ``                     |
+| 400 BAD_REQUEST | `Something Went Wrong` |
```
Comment (karthikmurakonda) [inline CDC_Backend/README.md:1123]: 404 NOT FOUND        `application not found`

**Review by karthikmurakonda — CHANGES_REQUESTED** [review]

**Review by karthikmurakonda — CHANGES_REQUESTED** [review]
.

File: CDC_Backend/README.md (hunk 5 above)
Comment (jp-karthik) [reply CDC_Backend/README.md:1297]: error added

File: CDC_Backend/README.md (hunk 3 above)
Comment (jp-karthik) [reply CDC_Backend/README.md:1278]: field added

File: CDC_Backend/README.md [hunk 6]
```diff
@@ -617,120 +602,113 @@ Response is a Json with these fields
 
 ```json
 {
-    "action": "Get Dashboard - Admin",
-    "message": "Data Found",
-    "ongoing": [
+  "action": "Get Dashboard - Admin",
+  "message": "Data Found",
+  "new": [],
```
[the remark this answers is not in the document]
Comment (jp-karthik) [reply CDC_Backend/README.md]: examples added

**Review by karthikmurakonda — APPROVED** [review]

**Pull request merged**